sql/patches/patch-revoke-permissions.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE /*$wgDBprefix*/mw_permissions
```

Comment:
I believe this file should be registered in `ManageWikiHooks::fnManageWikiSchemaUpdates`

Reply:
Fixed